filters/include/pcl/filters/filter_indices.h

```diff
@@ -109,7 +112,26 @@ namespace pcl
         deinitCompute ();
       }
 
-      /** \brief Set whether the regular conditions for points filtering should apply, or the inverted conditions.
+      /** \brief Calls the filtering method with specified executor and returns the filtered point cloud indices.
+        * \param[int] exec the executor to run the filter using
+        * \param[out] indices the resultant filtered point cloud indices
+        */
+      template <typename Executor>
+      void
+      filter (const Executor &exec, std::vector<int> &indices)
+      {
+        static_assert(!std::is_same<DerivedFilter, void>::value, "An executor overload doesn't exist.");
```

Comment:
should you not be checking directly if `DerivedFilter::applyFilter(exec, indices)` exists and is invocable?

Reply:
Fixed in 18659bb